node_virsh.sh

```diff
@@ -6,7 +6,8 @@ DIRECTIVE=${1-}
 
 set -u
 
-NODESXML=`ls node*.xml`
+CURRENT_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
The "pwd" builtin adds no information beyojnd BASH_SOURCE[0].   What do you think it is doing?  The two extra subshells seem like overkill.